5 - Post DESeq2 filtering and result annotation.Rmd

```diff
@@ -75,10 +84,16 @@ l2fc_cutoff <- 1
 
 # Only retain genes that are above a 0.2 CPM, below 0.05 for their adjusted p-value
 # and are differentially expressed greater than a Fold Change of 2 (L2FC = 1).
-results_annotated_sigDEGS <- results_annotated %>% dplyr::filter(padj < padj_cutoff & abs(log2FoldChange) > l2fc_cutoff)
-View(results_annotated_sigDEGS)
+RNA_seq_DE_annotated_sigDEGS <- RNA_seq_DE_annotated %>% dplyr::filter(padj < padj_cutoff & abs(log2FoldChange) > l2fc_cutoff)
```

Comment:
This retains genes that have padj < 0.05 and L2FC = 1, not genes with > 0.2CPM. Maybe move # Only retain genes that are > 0.2 CPM down to above `RNA_seq_DE_annotated_sigDEGS_cpm_filtered <- RNA_seq_DE_cpm_filtered_annotated %>% dplyr::filter(
  padj < padj_cutoff 
  & abs(log2FoldChange) > l2fc_cutoff
  )` ?

Reply:
Nice, good catch in the comments. I relabelled them.